### DISCUSSION TITLE
"Error Connecting to API"?

### DISCUSSION THREAD

**Discussion by lalonde302** [supabase/supabase#1835] [Questions] [answered]
I am making SELECT and UPDATE API requests to a table of level data from my mobile game.  All was working during testing when it was just myself and some playtesters.  

When the game rolled out to hundreds of players today, the supabase table became unresponsive and is only returning an API error : "TypeError : Failed to fetch".

I assume this was caused by the large influx of requests, but I was hoping this wouldn't be problem because there is no API request limit.  Either way, any help would be appreciated and let me know if I need to provide more information.  

Here's a screenshot from the Supabase table editor view:

![Screen Shot 2021-06-05 at 10 42 52 AM](https://user-images.githubusercontent.com/59236139/120897328-242f5400-c5eb-11eb-91da-08bd58b85064.png)

Is there anything I can do to recover the database?

**Comment by ru88s** [comment]
![スクリーンショット 2021-06-06 11 25 46](https://user-images.githubusercontent.com/42270511/120910591-121ce680-c6bb-11eb-8586-9cde6d7b6ec0.png)

I have a similar problem.

**Comment by inian** [comment, marked as answer]
Hi @lalonde302, please email beta@supabase.io with your project reference (in the URL). Thanks

**Reply by inian** [reply]
@ru88s and @lalonde302, hopefully both your issues are resolved now

**Reply by ru88s** [reply]
It has recovered.
Thank you for responding!

@lalonde302, Tank you for the question.
